torch_xla/csrc/runtime/pjrt_computation_client.cc

```diff
@@ -109,13 +109,18 @@ PjRtComputationClient::PjRtComputationClient() {
     bool async = sys_util::GetEnvBool(env::kEnvPjrtAsyncGpuClient, true);
     int local_process_rank = sys_util::GetEnvInt(env::kEnvPjRtLocalRank, 0);
     int global_process_rank = sys_util::GetEnvInt("RANK", local_process_rank);
+    int local_world_size = sys_util::GetEnvInt("LOCAL_WORLD_SIZE", 1);
+    int global_world_size = sys_util::GetEnvInt("WORLD_SIZE", local_world_size);
     std::string master_addr =
-        runtime::sys_util::GetEnvString("MASTER_ADDR", "localhost");
+        runtime::sys_util::GetEnvString("MASTER_ADDR", "127.0.0.1");
```

Comment:
cc @vanbasten23, it looks like our CI hosts don't support ipv6, but they resolve `localhost` to `::1`:

```
"os_error":"Address family not supported by protocol","syscall":"socket","target_address":"[::1]:8547"
```

https://github.com/pytorch/xla/actions/runs/6673957659/job/18140734664?pr=5733